src/status_im/chat/models/images.cljs

```diff
@@ -119,13 +121,23 @@
 
 (fx/defn camera-roll-get-photos
   {:events [:chat.ui/camera-roll-get-photos]}
-  [_ num]
-  {::camera-roll-get-photos num})
+  [_ num end-cursor]
+  {::camera-roll-get-photos [num end-cursor]})
+
+(fx/defn camera-roll-loading-more
+  {:events [:chat.ui/camera-roll-loading-more]}
+  [{:keys [db]} is-loading]
+  {:db (assoc db :camera-roll-loading-more is-loading)})
 
 (fx/defn on-camera-roll-get-photos
```

Comment:
We have ways of testing event handlers and there are examples in the codebase, I would say we should almost always test them. One of the principles of the https://deviq.com/principles/boy-scout-rule is to try to at least not increase tech debt, and if possible reduce them.